kubernetes/template/DeploymentTemplates/MainTemplate.json

```diff
@@ -261,38 +254,45 @@
         "description": "Container runtime to deploy on each cluster node."
       },
       "type": "string"
+    },
+    "localAKSeBinaryURL": {
+      "type": "string",
+      "defaultValue": "",
+      "metadata": {
+        "description": "Azure Stack blob url to download AKS engine from a blob store in disconnected environment."
+      }
+    },
+    "dvmImagePublisher": {
```

Comment:
Why do we want to expose this to customer? I think we should release every month along with AKS Engine + VHD and hardcode this value

Reply:
It is not exposed to the customer. But is available for testing internally. There are hard coded values there as well but can be overridden